metadata.yaml

```diff
@@ -63,7 +63,11 @@ provides:
     description: |
       Forwards the built-in grafana dashboard(s) for monitoring traefik.
     interface: grafana_dashboard
-
+  tracing:
```

Comment:
I forget, does the server must be the provider?
If yes, then it seems like traefik would need to be the requirer side?

Reply:
no it's correct like this. 
However I also get it mixed up all the time, we should consider changing the directionality.